packs/elastic/actions/show.snapshots.yaml

```diff
@@ -0,0 +1,82 @@
+---
+description: Show snapshots
+enabled: true
+entry_point: curator_runner.py
+name: show.snapshots
+parameters:
+  action:
+    default: show.snapshots
+    immutable: true
+    type: string
+  all_snapshots:
+    description: Do not filter snapshots.  Act on all snapshots.
+    type: boolean
+  dry_run:
+    default: false
+    description: Do not perform any changes.
+    type: boolean
+  exclude:
+    description: Snapshot list to operate on.
+    items:
+      type: string
+    type: array
+  host:
+    description: Elasticsearch host.
+    required: true
+    type: string
+  http_auth:
+    description: 'Use Basic Authentication ex: user:pass'
+    type: string
+  log_level:
```

Comment:
I'd pleased (it's nice to know) if you guys point me to a place where all the types are coded, since of course I couldn't find it docs. Maybe I'm as always too distracted and missed it:)